bin/start_test_migration

```diff
@@ -1,45 +1,55 @@
 #!/bin/bash
 
-echo "building dbs2go"
-make build
-
-# create pid file
-rm ./test-pids
-touch ./test-pids
-
-pwd
-
-# Start dbs-one-reader
+echo $LD_LIBRARY_PATH
```

Comment:
Dennis, please remove line 3 to line 11 (the build section) which should not be part of the unit test suit. This will automatically resolve issue with DYLD, LD paths. We can locally and manually build dbs2go executable while start test migration script will only use it